src/test/java/hudson/plugins/groovy/GroovyPluginTest.java

```diff
@@ -36,6 +37,7 @@ public void assignNullToBindingVariables() throws Exception {
 
     /// Roundtrip
 
+    @Ignore
```

Comment:
```
java.lang.AssertionError: 
Property oldScript is different
Expected: is null
     but: was "println 'Test'"
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualBeans(JenkinsRule.java:1807)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualProperties(JenkinsRule.java:1879)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualDataBoundBeans(JenkinsRule.java:1842)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualProperties(JenkinsRule.java:1871)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualDataBoundBeans(JenkinsRule.java:1831)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualProperties(JenkinsRule.java:1871)
	at org.jvnet.hudson.test.JenkinsRule.assertEqualDataBoundBeans(JenkinsRule.java:1831)
	at hudson.plugins.groovy.GroovyPluginTest.roundtripTestSystemGroovyStringScript(GroovyPluginTest.java:47)
```

(Note that the arguments to `assertEqualDataBoundBeans` seem to be backwards.)

Perhaps due to https://github.com/jenkinsci/script-security-plugin/commit/afb290b4bd3414d292890a5f09333ef04c4fb820? @daniel-beck @jtnord is the change in behavior here expected, and if so, is there a recommend update for test idioms or just ignoring the test?

Reply:
I would not expect that any beans / serialisation / round trip behave differently, I would merely expect that the script is not approved.
That - said if the test is not using any security in Jenkins (AuthN/AuthZ) which it does not appear to be, then the script should be approved regardless of `SECURITY-3103` because of [this check](https://github.com/jenkinsci/script-security-plugin/blob/23895f409fbdbd3419565e2530a6c29a283ca399/src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java#L602-L608)

Reply:
Likely caused by SECURITY-2450, see https://github.com/jenkinsci/extensible-choice-parameter-plugin/pull/67 for a fix for same issue in `extensible-choice-parameter`.

Reply:
OK, I will try that when I get a moment.